cpp/include/IceGrid/PluginFacade.h

```diff
@@ -60,7 +60,7 @@ namespace IceGrid
     class RegistryPluginFacade
```

Comment:
There is no ICEGRID_API decoration because the implementation is provided by the icegridregistry - that loads your plugin - not by a shared library you link with.